xmtp_mls/migrations/2025-01-16-143131_add_message_expiration_to_groups/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE GROUPS ADD COLUMN message_expire_from_ms  BIGINT NOT NULL DEFAULT 0;
```

Comment:
settings this to nullable causes some issues in using `between` in diesel, for now decided to go like this, but will try to resolve it and make this nullable.

Reply:
is the between usage problematic only in the webassembly build, or both wasm and native?

Reply:
Resolved it in my latest update!